homework_30_11/src/Triangle/Triangle.java

```diff
@@ -0,0 +1,89 @@
+package Triangle;
+import java.util.Scanner;
+/*Написать класс Triangle имитирующий прямоугольный треугольник, со сторонами a,b,c и методами получения величины каждой
+    стороны, площади, периметра треугольника. а так же добавить метод проверяющий выполнение теоремы пифагора.
+    Если стороны заданы не верно вывести сообщение на екран.*/
+public class Triangle {
+    double sideA;
+    double sideB;
+    double sideC;
+
+    public Triangle(double sideA,double sideB,double sideC) {
+        this.sideA=sideA;
+        this.sideB=sideB;
+        this.sideC=sideC;
+    }
+
+   /* public double getSideA() {
+        Scanner inputScanner = new Scanner(System.in);
+        double sideA = inputScanner.nextDouble();
+        System.out.print("Please, enter side A of the triangle, sm: ");
+        return getSideA();
+    }
+
+    public double getSideB() {
+        Scanner inputScanner = new Scanner(System.in);
+        double sideB = inputScanner.nextDouble();
+        System.out.print("Please, enter side B of the triangle, sm: ");
+        return getSideB();
+    }
+
+    public double getSideC() {
+        Scanner inputScanner = new Scanner(System.in);
+        System.out.print("Please, enter side C of the triangle, sm: ");
+        double sideC = inputScanner.nextDouble();
+        return getSideC();
```

Comment:
in all getSide methods you need to return a value of side

Reply:
It was the first solution, i commented it, and did the other way.

Reply:
you still need this methods, to get needed side value